src/CheckoutSdk.Extensions/CheckoutServiceCollectionExtensions.cs

```diff
@@ -7,10 +7,10 @@
 
 namespace CheckoutSDK.Extensions.Configuration
 {
-    public static class CheckoutServiceCollection
+    public static class CheckoutServiceCollectionExtensions
```

Comment:
What is the purpose of renaming to `Extensions`?
Unfortunately, this might cause breaking changes for other merchants, which is not cool, if we can keep the signature as it this for now, that would be great.